### PR TITLE
fix: `spawn` not working

### PR DESCRIPTION
**Pull request by orgrimarr**
Fixes https://github.com/vercel/pkg/issues/1356

### CODE REVIEW

**Review by robertsLando — COMMENTED** [review]
Instead of using an env var I suggest to auto-detect the spawn of itself in `childProcess.spawn` / `childProcess.spawnSync` calls, it makes more sense. Also if you want to test locally just run the single test you want not all the others. You pushed a huge pnpm-loc.yaml file, remove it

**Review by robertsLando — CHANGES_REQUESTED** [review]
Instead of using an env var I suggest to auto-detect the spawn of itself in `childProcess.spawn` / `childProcess.spawnSync` calls, it makes more sense. Also if you want to test locally just run the single test you want not all the others. You pushed a huge pnpm-loc.yaml file, remove it

File: .gitignore [hunk 1]
```diff
@@ -20,4 +20,8 @@ examples/express/node_modules
 # Editors
 .vscode/
 yarn-error.log
-tsconfig.tsbuildinfo
\ No newline at end of file
+tsconfig.tsbuildinfo
+
+# Tests temp
+*.exe
+*.bak
```
Comment (robertsLando) [inline .gitignore:27]: remove changes to this file

**Review by robertsLando — CHANGES_REQUESTED** [review]

**Comment by robertsLando** [comment]
Also fix lint issues please

File: .gitignore (hunk 1 above)
Comment (orgrimarr) [reply .gitignore]: Should be fixed now

**Review by robertsLando — CHANGES_REQUESTED** [review]
A test is still failing

**Comment by robertsLando** [comment]
Ok

**Comment by lamualfa** [comment]
This is great. Please merge it.

**Pull request closed without merging**